docs/modules/operation/pages/deep-dive/alarms/history.adoc

```diff
@@ -37,49 +31,46 @@ admin@opennms()> config:property-set elasticUrl http://es:9200
 
- feature:install opennms-alarm-history-elastic
-
-[[ga-alarm-history-indexing]]
 == Alarm indexing
 
-When alarms are initially created, we push a document to Elasticsearch that includes all of the alarm fields as well as additional details on some of the related objects (i.e., the node.)
-
-In order to avoid pushing a new document every time a new event is reduced on to an existing alarm, we only push a new document when (at least) one of these conditions are met:
+When alarms are created, a document that includes all alarm fields, and additional details on related objects (for example, the node), is pushed to Elasticsearch.
+To avoid pushing a new document every time a new event is reduced to an existing alarm, documents are pushed only when at least one condition is met:
 
-. We have not recently pushed a document for that alarm. (See `alarmReindexDurationMs`.)
-. The severity of the alarm has changed.
-. The alarm has been acknowledged or unacknowledged.
-. Either of the associated sticky or journal memos have changed.
-. The state of the associated ticket has changed.
-. The alarm has been associated with, or removed, from a situation.
-. A related alarm has been added or removed from the situation.
+* A document has not recently been pushed for the alarm (see `alarmReindexDurationMs`).
+* The severity of the alarm has changed.
+* The alarm has been acknowledged or unacknowledged.
+* The associated sticky or journal memos have changed.
+* The state of the associated ticket has changed.
+* The alarm has been associated with or removed from a situation.
+* A related alarm has been added to or removed from the situation.
 
-NOTE: To change this behavior and push a new document for *every* change, you can set `indexAllUpdates` to `true`.
+NOTE: To change this behavior and push a new document for every change, you can set `indexAllUpdates` to `true`.
 
-When alarms are deleted, we push a new document that contains the alarm id, reduction key, and deletion time.
+When alarms are deleted, a new document containing the alarm ID, reduction key, and deletion time is pushed to Elasticsearch.
 
-The following table describes a subset of the fields in the alarm document:
+=== Alarm document fields
 
-[options="header"]
-[cols="1,3"]
+[caption=]
+.Alarm document fields
+[options="autowidth"]
 |===
 | Field | Description
 
 | @first_event_time
-| Timestamp in milliseconds associated with the first event that triggered this alarm.
+| Time (in milliseconds) associated with the first event that triggered this alarm.
 
 | @first_event_time
-| Timestamp in milliseconds associated with the last event that triggered this alarm.
+| Time (in milliseconds) associated with the last event that triggered this alarm.
 
 | @update_time
-| Timestamp in milliseconds at which the document was created.
+| Time (in milliseconds) when the document was created.
```

Comment:
Because alarms describe scenarios that may span hours or days, it's often quite helpful to know both the create time and the update time for a given alarm.